openfecwebapp/templates/layouts/main.html

```diff
@@ -68,18 +67,56 @@
   <main id="main" {% if section %} data-section="{{section}}"{% endif %}>
     {% block body %}{% endblock %}
   </main>
+
+<footer class="footer-links">
```

Comment:
That's a good idea. It's in the footer area but it is a set of links, just near the footer. I figure the class name will be descriptive enough as well.